packages/components/Checkbox/src/CheckboxTokens.win32.ts

```diff
@@ -12,7 +12,7 @@ export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: T
       checkboxBorderWidth: globalTokens.stroke.width.thin,
       checkboxBorderRadius: globalTokens.corner.radius.small,
       checkboxSize: 16,
-      checkmarkSize: 8,
+      checkmarkSize: 12,
```

Comment:
Should we change the base CheckboxTokens.ts file as well since the SVG is shared?

Also, hook em!

Reply:
I see no reason not to. Give me a second to push another commit.